spec/tasks/whitehall_importer_spec.rb

```diff
@@ -5,6 +5,13 @@
 
   let(:import_data) { whitehall_export_with_one_edition }
 
+  before do
+    binary_image = File.open(File.join(fixtures_path, "files", "960x640.jpg"), "rb").read
```

Comment:
It looks like we can simplify this (and similar) to `file_fixture("960x640.jpg").read` - [`file_fixture`](https://relishapp.com/rspec/rspec-rails/v/3-8/docs/file-fixture) returns a path